src/test/java/org/tarantool/TarantoolControl.java

```diff
@@ -216,44 +193,67 @@ public void waitStarted(String instanceName) {
     }
 
     /**
-     * Wait until the instance will be stopped.
+     * Sends <code>stop</code> command using tarantoolctl.
      *
-     * Use tarantoolctl status instanceName.
+     * @param instanceName target instance name
      */
-    public void waitStopped(String instanceName) {
-        while (status(instanceName) != 1) {
-            sleep();
+    public void stop(String instanceName) {
```

Comment:
I understood why we need start() w/o waiting in the public API: to bootstrap a full-mesh replication cluster (tarantool does not exit from box.cfg() until the cluster will be built). But I don't see cases to async stop() and I feel it very error-prone to have it in the public API.

I would also wrap waitStarted() into start() and add an optional argument (say, `wait`) to disable it (and so we need provide waitStarted() as public too). This way will push us to start an instance asyncronously only when it is intentional.